gke-deploy/VENDOR-LICENSE

```diff
@@ -1,4 +1,211 @@
 
+===========================================================
+Import: github.com/GoogleCloudPlatform/cloud-builders/gke-deploy/vendor/github.com/GoogleCloudPlatform/cloud-builders/gke-deploy
```

Comment:
This indicates you're vendoring yourself somewhere, which you definitely don't want to be doing.

Reply:
This must've started happening because of generate-docs go binary.